spk/shairport/src/dsm-control.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+
+# Others
+SHAIRPORT="${SYNOPKG_PKGDEST}/bin/shairport"
+VAR_DIR="${SYNOPKG_PKGDEST}/var"
```

Comment:
Add the var dir to the package like this: https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/Makefile#L40. 
Then, your PID_FILE might be defined as done here: https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/src/dsm-control.sh#L14